pages/update-content/[id].js

```diff
@@ -0,0 +1,63 @@
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import Form from "~/components/Form";
+import Layout from "~/components/Layout/Layout";
+
+const UpdateContent = () => {
+  const router = useRouter();
+  const { id } = router.query;
+  const { data: issueContent, error: issueContentError } = useSWR(
+    `/api/content/${id}`
+  ); // fetch content related to an issue
+
+  const handleUpdateContent = async (data) => {
+    const response = await fetch(`/api/content/${id}`, {
+      method: "PUT",
+      headers: {
+        "Content-Type": "application/json",
+        Accept: "application/json",
+      },
+      body: JSON.stringify(data),
+    });
+
+    if (response.ok) {
+      router.push(`/issues/${data.relations}`);
+    } else {
+      setError(true);
```

Comment:
Maybe I'm missing something. Where does this `setError` function get declared? What is it doing?